SPECS/WALinuxAgent/0002-fix-bump-version-to-2.11.8.8.patch

```diff
@@ -0,0 +1,30 @@
+From e2f24e98cd7f495d5cf6ef436625d9dad22785eb Mon Sep 17 00:00:00 2001
+From: Chris Co <chrco@microsoft.com>
+Date: Sat, 3 Aug 2024 22:25:04 -0700
+Subject: [PATCH] fix: bump version to 2.11.8.8
+
+In order to prevent further 2.11 agent upgrades, bump the version to
+2.11.8.8 so any upcoming 2.11 hotfixes are not applied. 2.12 will be
+released soon and the agent will take the 2.12 upgrade correctly.
+
+Signed-off-by: Chris Co <chrco@microsoft.com>
+---
+ azurelinuxagent/common/version.py | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/azurelinuxagent/common/version.py b/azurelinuxagent/common/version.py
+index c1715ff7..b4e93e81 100644
+--- a/azurelinuxagent/common/version.py
++++ b/azurelinuxagent/common/version.py
+@@ -209,7 +209,7 @@ AGENT_LONG_NAME = "Azure Linux Agent"
+ #
+ #            When doing a release, be sure to use the actual agent version. Current agent version: 2.4.0.0
+ #
+-AGENT_VERSION = '2.11.1.4'
++AGENT_VERSION = '2.11.8.8'
```

Comment:
Is `2.11.8` a value that we read from a dev branch or future WALinuxAgent version release plan? Based on my understanding of reading the comment, this version should represent the last minor version of v2.11 to prevent updating to a higher v2.11 version at runtime. Please correct me if misunderstood here.

Reply:
Yes, your understanding is correct. 2.11.8 will be higher than any agent version hotfix for 2.11. And the WALinuxAgent will be updating to 2.12 in their next release which is upcoming

Reply:
Thanks for the clarification!